src/test/java/ibm/jceplus/junit/openjceplusfips/integration/TestTLS.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright IBM Corp. 2024
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution.
+ */
+package ibm.jceplus.junit.openjceplusfips.integration;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import ibm.jceplus.junit.base.integration.BaseTestTLS;
+
+public class TestTLS extends BaseTestTLS{
+
+    @BeforeAll
+    public static void init() throws Exception {
+        if (java.security.Security.getProvider("OpenJCEPlus") != null) {
+            java.security.Security.removeProvider("OpenJCEPlus");
+        }
```

Comment:
Add a check to ensure the openjceplus provider is not included when running the openjceplusfips integration test.

Reply:
resolved